drivers/scheduler/k8s/k8s.go

```diff
@@ -245,9 +245,10 @@ func (k *k8s) Schedule(instanceID string, options scheduler.ScheduleOptions) ([]
 }
 
 func (k *k8s) createStorageObject(spec interface{}, ns *v1.Namespace, app *spec.AppSpec) (interface{}, error) {
+	k8sOps := k8sops.Instance()
```

Comment:
Rename the variable to something else? A bit confusing whether using the package or the instance.
Or you could just use k8sops.Instance() everywhere.